### PR TITLE
[JBPM-8699] Fix exception handling in ClassCacheManager

### PR DESCRIPTION
**Pull request by Hesoyam2**
Fix exception handling

### CODE REVIEW

**Comment by kie-ci10** [comment]
Can one of the admins verify this PR? Comment with 'ok to test' to start the build.

File: jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ClassCacheManager.java [hunk 1]
```diff
@@ -57,7 +57,7 @@ public Command findCommand(String name, ClassLoader cl) {
                     Command commandInstance = (Command) Class.forName(name, true, cl).newInstance();
                     commandCache.put(name, commandInstance);
                 } catch (Exception ex) {
-                    throw new IllegalArgumentException("Unknown Command implementation with name '" + name + "'");
+                    throw new IllegalArgumentException("Unknown Command implementation with name '" + name + "': " + ex.getMessage());
```
Comment (mswiderski) [inline jbpm-services/jbpm-executor/src/main/java/org/jbpm/executor/impl/ClassCacheManager.java]: should we also add it as the cause of the `IllegalArgumentException`?

**Comment by mswiderski** [comment]
ok to test

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Pull request merged**